modules/hosted-restart-hcp-components.adoc

```diff
@@ -22,9 +22,12 @@ $ oc annotate hostedcluster \
 
 .Verification
 
-The control plane is restarted whenever the value of the anonotation changes. The `date` command in the example serves as the source of a unique string. The annotation is treated as a string, not a timestamp.
+The control plane is restarted whenever the value of the annotation changes. The `date` command in the example serves as the source of a unique string. The annotation is treated as a string, not a timestamp.
 
-The following components are restarted:
+[NOTE]
+====
+After you restart a control plane, the following {hcp} components are typically restarted. However, if you implement any change in one of the components, as a side effect, other components might also restart.
```

Comment:
I don't think this is accurated:

"if you implement any change in one of the components, as a side effect, other components might also restart"

I would say something like:

“However, any changes made by other components, as a side effect, could cause the restart of additional components.”

Reply:
Addressed

Reply:
@jparrill this PR can be merged from 4.14-4.18, right?